website/docs/reference/resource-limits.mdx

```diff
@@ -3,8 +3,8 @@ title: Resource Limits
 ---
 import Figure from '@site/src/components/Figure/Figure.tsx'
 
-:::info Availability
-Resource limits are currently in beta for hosted customers.
+:::note Availability
+**Plan**: All plans | **Version**: `6.2+`
```

Comment:
Yeah, I'm down with that. I've only seen the enterprise-only updates you've done, so I didn't know what to do. I don't mind removing it, but does that make it less clear? Because it might be that you think we forgot to specify it? I like being explicit, so I might lean towards specifying it, but no strong feelings either way

Reply:
Hmm, but then again, that only applies to pages / sections where we talk about availability 🤷🏼  I dunno.

Reply:
Yes, I think the way we don't mention on, for instance, the feature flags page that it's available for all plans, we don't need to specify it unless there is a restriction. I'll create a page on availability, version, beta, etc. that we can link to and explain in a bit more detail what these concepts mean.